code/go/0chain.net/miner/miner_test.go

```diff
@@ -147,37 +169,40 @@ func TestBlockVerificationBadHash(t *testing.T) {
 	common.Done()
 }
 
-func TestBlockVerificationTooFewTransactions(t *testing.T) {
-	SetUpSingleSelf()
-	ctx, clean := getContext()
-	defer clean()
-	mr := CreateRound(1)
-	b, err := generateSingleBlock(ctx, nil, mr)
-	if err != nil {
-		t.Errorf("Error generating block: %v", err)
-		return
-	}
-	mc := GetMinerChain()
-	txnLength := numOfTransactions - 1
-	b.Txns = make([]*transaction.Transaction, txnLength)
-	if b != nil {
-		for idx, txn := range b.Txns {
-			if idx < txnLength {
-				b.Txns[idx] = txn
-			}
-		}
-		_, err = mc.VerifyRoundBlock(ctx, mr, b)
-	}
-	if err == nil {
-		t.Error("FAIL: Block with too few transactions passed verification")
-	} else {
-		t.Log("SUCCESS: Block with too few transactions failed verifcation")
-	}
-	common.Done()
-}
+//func TestBlockVerificationTooFewTransactions(t *testing.T) {
```

Comment:
Can we skip this test case instead of commenting it out?
If yes, this is preferred since the code is statically checked and would degrade slower.
Commented out code easily becomes invalid after some time.